### DISCUSSION TITLE
[求助] nonebot2识别不到本地插件

### DISCUSSION THREAD

**Discussion by lonelyang** [nonebot/discussions#508] [问答]
初始化后加载了内置插件就直接运行了
![image](https://github.com/user-attachments/assets/5362b6b5-78ed-446d-9805-2317236f88d0)
![image](https://github.com/user-attachments/assets/28e6973a-0969-423d-8bac-ddf02980b2d0)
![image](https://github.com/user-attachments/assets/baf9e0e6-f19f-4c71-a4c0-3c8ccd8bc96f)

**Comment by yanyongyu** [comment]
看不出来有啥问题，你的运行目录正确吗
